charms/worker/k8s/scripts/update_alert_rules.py

```diff
@@ -100,28 +100,25 @@ def move_processed_files(temp_dir):
     for temp_file in temp_dir.iterdir():
         final_path = ALERT_RULES_DIR / temp_file.name
         shutil.move(str(temp_file), str(final_path))
-        logging.info(f"Moved {temp_file.name} to {final_path}")
+        logging.info("Moved %s to %s", temp_file.name, final_path)
 
 
 def apply_patches():
     """Apply patches to the downloaded and processed rule files."""
     for patch_file in PATCHES_DIR.glob("*"):
-        logging.info(f"Applying patch {patch_file}")
+        logging.info("Applying patch %s", patch_file)
         subprocess.check_call(["/usr/bin/git", "apply", str(patch_file)])
 
 
 def main():
     """Fetch, process, and save AlertManager rules."""
     with TemporaryDirectory() as temp_dir:
         temp_path = Path(temp_dir)
-        try:
-            download_and_process_rule_files(temp_path)
-            shutil.rmtree(ALERT_RULES_DIR, ignore_errors=True)
-            ALERT_RULES_DIR.mkdir(parents=True)
-            move_processed_files(temp_path)
-            apply_patches()
-        except Exception as e:
-            logging.error("An error occurred: %s" % e)
+        download_and_process_rule_files(temp_path)
+        shutil.rmtree(ALERT_RULES_DIR, ignore_errors=True)
+        ALERT_RULES_DIR.mkdir(parents=True)
+        move_processed_files(temp_path)
+        apply_patches()
```

Comment:
@mateoflorido looks like we used to blind swallow all exceptions here and log them.  That seem... not great in practice.  

pylint hated it too -- so i just removed it.  Do you know of any reason we'd swallow all exceptions?

Reply:
this isn't even part of the charm is it?  It's like a CI script right?

Reply:
That's correct, this a maintenance script which updates the rules for AlertManager.